contrib/mongodb/mongo-go-driver/mongo/scrub.go

```diff
@@ -0,0 +1,50 @@
+package mongo
+
+import "github.com/mongodb/mongo-go-driver/bson"
+
+func scrub(doc *bson.Document) *bson.Document {
```

Comment:
This is a very good solution to scrubbing Mongo. FWIW we've started doing all of that in the agent already (see [here](https://github.com/gbbr/datadog-trace-agent/blob/master/obfuscate/obfuscate.go#L53)) but it's currently addressing JSON, which in most cases will be invalid with Mongo. The implementation is not fully developed. I'd be curious to find out what our other tracers are sending to the agent and if we could easily decode BSON to JSON for obfuscation. This should be pretty quick, and even if we don't decide to have it by default, it's nice to have at least an opt-in.

@palazzem do we want to obfuscate at integration level? This is a nice solution (and I assume it's fast), we could keep it and try using something similar in the agent.

@dd-caleb can we benchmark this?